src/test/java/htsjdk/samtools/CRAMAllEncodingStrategiesTest.java

```diff
@@ -108,13 +155,17 @@ public void assertRoundTripFidelity(
         }
     }
 
-    private void assertRoundtripFidelityWithSamtools(final File sourceCRAM, final File referenceFile) throws IOException {
+    private void assertRoundtripFidelityWithSamtools(
+            final File sourceCRAM,
+            final File referenceFile,
+            final boolean lenientEquality,
+            final boolean emitDetail) throws IOException {
         if (SamtoolsTestUtils.isSamtoolsAvailable()) {
```

Comment:
If samtools is not available, the test probably shouldn't just silently do nothing and succeed. Maybe throw a TestNG `SkipException` in that case?

Reply:
Makes sense. Done.